.github/workflows/discord-release-announcement.yaml

```diff
@@ -0,0 +1,18 @@
+name: Discord Release Announcement
+
+on:
+  release:
+    types: [prereleased, published]
+
+jobs:
+  delimiter-test:
+    runs-on: ubuntu-latest
+    name: Send message to discord
+    steps:
+      - name: parse and send message
+        uses: michelengelen/discord-message-action@02af30a15955ecf718049bc33b0efabf6f626e0b
```

Comment:
By the way, maybe we should review the action itself? Since it's the most essential part of the code.

Reply:
This is the link to it for reference: https://github.com/michelengelen/discord-message-action

Reply:
I checked it - seems good. I think we can move forward and add this to the release workflow